rdrf/report/templates/report_designer.html

```diff
@@ -209,37 +225,50 @@ <h1>
         </div>
     </fieldset>
     <fieldset>
-        <legend>Patient Filters</legend>
-        <div class="row">
-            <label for="filter_working_groups" class="col-sm-2 col-form-label">{{ form.filter_working_groups.label }}</label>
-            <div class="col-sm-10 mt-2">
-                {{ form.filter_working_groups }}
-            </div>
-        </div>
-        <div class="row">
-            <label for="filter_consents" class="col-sm-2 col-form-label">{{ form.filter_consents.label }}</label>
-            <div class="col-sm-10 mt-2">
-                <ul id="id_filter_consents_list" class="list-unstyled border p-2" style="max-height: 200px; overflow: auto">
-                {% for value, label in form.filter_consents.field.choices %}
-                    {% if value %}
-                    <li>
-                        <label class="form-check-label">
-                            <input type="checkbox" name="filter_consents" value="{{ value }}"
-                                {% if value in form.filter_consents.initial %}
-                                    checked="checked"
-                                {% endif %}
-                            >
-                            {{ label }}
-                        </label>
-                    </li>
-                    {% endif %}
-                {% endfor %}
-                </ul>
-            </div>
-            <div class="offset-sm-2 col-sm-10 form-text">
-                {% trans "Filter the report to only include patients that have consented to the selected consent items." %}
-            </div>
-        </div>
+        <legend>Report Filters</legend>
+        <ul class="list-group offset-sm-2 col-sm-10">
+            <li class="list-group-item">
+                {{ form.filter_by_working_groups|add_attr:"class,form-check-input report-filter-toggle" }}
+                <label for="id_filter_by_working_groups">{{ form.filter_by_working_groups.label }}</label>
+                <div class="row d-none report-filter">
+                    <label for="id_filter_working_groups" class="visually-hidden">{{ form.filter_working_groups.label }}</label>
+                    <div class="mt-2">
+                        {{ form.filter_working_groups }}
+                    </div>
+                    <div class="form-text">
+                        {% trans "Filter the report to only include patients within the selected working groups." %}
```

Comment:
Maybe this should say:

> Filter the report to only include patients within **one of** the selected working groups.

And the other should say:

> Filter the report to only include patients that have consented to **all of** the selected consent items.

(without the bold)

Reply:
Good point.

Reply:
Updated.